### DISCUSSION TITLE
Problems under PHP 7.4

### DISCUSSION THREAD

**Discussion by BlackFlag69** [codeproducer198/Runalyze#2] [General]
Hello,

it seems, that you are the only one who fixes some bugs in runalyze. I also use runalyze on my own server, but since I updated the server (Debian) and PHP to 7.4 I can not import activities anymore. I tried to debug it, but had have not even found the error. Have you ever tried to run it under 7.4 and if so, can you give me a hint how to solve my problem?
Thanks in advance

**Comment by codeproducer198** [comment]
Hi Black,
i have only PHP 7.3 running, but have upgraded now some PHP packages in preparation for PHP > 7.3 - but never (yet) tried 7.4. 
I don't know if this resolves your problem - i think not.
Whats the exact problem? If you don't get an error or 500 it must happened something. Have you done a look at the logs under var/logs?
A common approach for me is to run the app (locally) in the developer mode - so are more log outputs available.
To enable the dev-mode i do this on my env:
> In der `/etc/nginx/sites-available/nginxRunalyze` die Zeile anpassen 
> - `try_files $uri /app.php$is_args$args;` -> `try_files $uri /app_dev.php$is_args$args;`
> - `location ~ ^/app\.php(/|$)` -> `location ~ ^/(app_dev|config)\.php(/|$)`.
>
